src/controllers/usersController.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const usersService = require('../services/usersServices');
+
+beforeEach(() => {
+  usersService.deleteAllUsers();
+});
+
+const getUsers = (req, res) => {
+  res.send(usersService.getUsers());
+};
+
+const getUser = (req, res) => {
+  const { id } = req.params;
+  const user = usersService.getUser(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.status(200).send(user);
+};
+
+const createUser = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  res.status(201).send(usersService.createUser(name));
+};
+
+const deleteUser = (req, res) => {
+  const { id } = req.params;
+
+  if (!usersService.getUser(+id)) {
```

Comment:
Better to create constants for such logic `!usersService.getUser(+id)` and then use this variable
The same in all other places in code